src/lotus/types.rs

```diff
@@ -0,0 +1,196 @@
+use cid::Cid;
```

Comment:
Please rename this file to something less generic. Using a `types.rs` file is considered an antipattern because it quickly degenerates into a dumping ground for random declarations without much concern for proper structure.

This file seems to be mostly dedicated to declaring message types used for the JSON-RPC methods. Please let it reflect that.